res/values/strings.xml

```diff
@@ -52,4 +52,7 @@
     <string name="warning_nothing_to_send">Nothing to send</string>
     <string name="mail_queued">Mail will be send when online</string>
 
+    <string name="files_shared_1">File: %s</string>
+    <string name="files_shared_2">Files: %s and %s</string>
+    <string name="files_shared_3">Files: %s and %d more</string>
```

Comment:
From my side, we can remove the `File(s):` prefix. However, the last string needs to be a quantity string, because different languages have different ways to say 1 more, 2 more, 3 more, etc.

Reply:
I tested having a suffix first, then tried prefix and realized it made it so much clearer that it's an attachment at first glance while browsing emails. 

Can you let me know your preferred format for 1, 2, and 3+ files? I'll update this pull request to exactly match your preference.

Thanks!

Reply:
If you think we need the suffix, I am fine with it. I just think that `files_shared_3` needs to be a quantity string.

Reply:
I've replaced files_shared_1/2/3 by a single plural string. It gives us a little less flexibility (we lose files_shared_2 because no quantity expresses 2 reliably) but it makes the code simpler and presumably easier to translate, I can see that now!